piker/clearing/_paper_engine.py

```diff
@@ -30,7 +30,6 @@
     Callable,
 )
 import uuid
-
```

Comment:
This is a subtle style thing but we're trying to keep imports to 3 *sections* separate by a single newline:
- stdlib mods
- 3rd party mods
- internal imports

so if you can add back this NL it'd be swell 👍🏼